srcpkgs/rofi-wayland/template

```diff
@@ -0,0 +1,35 @@
+# Template file for 'rofi-wayland'
+pkgname=rofi-wayland
+version=1.7.5+wayland1
+revision=1
+build_style=meson
+hostmakedepends="pkg-config git flex glib-devel wayland-devel"
+makedepends="libglib-devel cairo-devel pango-devel libxkbcommon-devel
+ gdk-pixbuf-devel wayland-devel libmpdclient-devel libnl3-devel libxcb-devel
+ xcb-util-devel xcb-util-wm-devel xcb-util-cursor-devel
+ startup-notification-devel wayland-protocols alsa-lib-devel check-devel"
+depends="rofi>=${version%+*} bash which sed"
+short_desc="Window switcher, run dialog and dmenu replacement - Wayland"
+maintainer="Subhaditya Nath <sn03.general@gmail.com>"
+license="MIT"
+homepage="https://github.com/lbonn/rofi"
+changelog="https://raw.githubusercontent.com/lbonn/rofi/wayland/Changelog"
+distfiles="https://github.com/lbonn/rofi/releases/download/${version%%+*}%2B${version##*+}/rofi-${version}.tar.gz"
+checksum=4b7b5de0606e91380861dce6f2b894bebf63b2031c8746fd4096f72576ce541a
+
+post_extract() {
+	for f in script/rofi-*; do
+		mv $f "$f-wayland"
+	done
+	for f in data/*; do
+		mv $f "${f%.*}-wayland.${f##*.}"
+	done
+	for f in doc/*.?; do
+		mv $f ${f%.?}-wayland.${f##*.}
+	done
```

Comment:
I think we should just make it conflict with rofi, but I am not really interested in merging this

Reply:
> I am not really interested in merging this

same here, i don't see a reason in packaging a fork when several alternatives/clones to rofi that support wayland are already in the repos

Reply:
Those alternatives and clones don't support the plugins that have been specifically written for rofi. (rofi-emoji, for example.)